apps/css-workshop/src/pages/anchor.astro

```diff
@@ -15,24 +16,37 @@ import Layout from './_layout.astro';
 
   <section id='demo-default'>
     <p>
-      <a href='https://itwin.github.io/iTwinUI/' class='iui-anchor' id='test-1'
-        >iTwinUI components list</a
-      > - A basic anchor link.
+      <Anchor_ href='https://itwin.github.io/iTwinUI/' id='test-1'>iTwinUI components list</Anchor_>
+       - A basic anchor link.
     </p>
 
     <p>
-      <a
-        href='https://www.bentley.com/en'
-        class='iui-anchor iui-anchor-external'
-        referrerpolicy='no-referrer'
-        target='_blank'
-        id='test-2'>Bentley Systems<span class='iui-visually-hidden'>(opens in new tab)</span></a
-      > - An anchor link that opens in a new tab / window.
+      <Anchor_ external='true' href='https://www.bentley.com/en'>
```

Comment:
This anchor is the one that should have the `id='test-2` set on it, not [this one](https://github.com/iTwin/iTwinUI/blob/97891d6ac107dc60547b9082b1c8496516dfa870/apps/css-workshop/src/pages/anchor.astro#L31). This change is breaking the hover and focus visual tests for this file.

Reply:
Moved `id="test-2"` from the anchor with `this one` as a label to `Bentley Systems`.